doc/developer/debugging.rst

```diff
@@ -164,11 +164,24 @@ Just define an environment variable in the build:
 Developing in Python
 --------------------
 
-Add the following environment variable to the geoportal container:
+Create a development docker-compose.overide.yml
+...............................................
 
-``GUNICORN_PARAMS="-b :80 --worker-class gthread --threads 1 --workers 1 --reload"``
+Add a ``docker-compose.override.yml`` file with a ``geoportal`` service containing the following lines:
+
+.. code:: yaml
+
+   ---
+
+   version: '2'
 
-TODO
+   services:
+     geoportal:
+       volumes:
+         - ${PWD}/geoportal/<project>_geoportal:/app/aprona_geoportal
+       environment:
+         - GUNICORN_CMD_ARGS=--reload
+       entrypoint: []  # do not eval templates when using webpack-dev-server
```

Comment:
For me this will not work If we don't use webpack-dev-server, because we need the interpreted templates...

Reply:
We might put a # in front of this line